SQL/0000-00-00-schema.sql

```diff
@@ -864,6 +864,7 @@ CREATE TABLE `document_repository` (
   `EARLI` tinyint(1) DEFAULT '0',
   `hide_video` tinyint(1) DEFAULT '0',
   `File_category` int(3) unsigned DEFAULT NULL,
+  `uuid` varchar(36) DEFAULT '',
```

Comment:
@PapillonMcGill recently set up a standard for all new additions to our SQL database (https://github.com/aces/Loris/blob/19.0-dev/docs/SQLModelingStandard.md). I think uuid should maybe be UUID? or something a bit more descriptive? melanie, what do you think?

Reply:
@zaliqarosli @intralizee 
The standards say that `The acronym should only contain upper case letter.` So capitalized is the way to go. UUID might not be fully known by neuroimaging community (yet) but software developer within knows it well.

Reply:
@intralizee i had to look up what UUID meant 😝 but i think its fine here